.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2
 # indicated by the CC/CXX environment variables.
 default_job: &default_job
   docker:
-    - image: cosmoepfl/rascal-ci:2
+    - image: cosmoepfl/rascal-ci:3
```

Comment:
What is the difference between 2/3 exactly?

Reply:
Apparently 2 existed while 3 does not according to the CI.

Reply:
The difference is https://github.com/cosmo-epfl/librascal/pull/237/files#diff-bebe822d715123ec31b03d977212fe2a> And I forgot to update the dockerhub image, I'll restart the build

Reply:
Everything is here: https://hub.docker.com/repository/docker/cosmoepfl/rascal-ci

Reply:
Hmm, I need a docker account to access it. I'll check that.